docs/docs/genai/eval-monitor/insights/analyze-experiment.mdx

```diff
@@ -0,0 +1,127 @@
+import Tabs from "@theme/Tabs"
+import TabItem from "@theme/TabItem"
+import { MessageSquare, Target, BarChart3, ExternalLink, Bot, Users, CheckCircle, TrendingUp, Database, Book, Rocket, Scale } from "lucide-react";
+
+# Analyze Experiment
+
+Automatically analyze traces in your MLflow experiments to identify operational issues, quality problems, and performance patterns. The Analyze Experiment tool uses hypothesis-driven analysis to systematically examine your GenAI application's behavior and generate comprehensive markdown reports.
+
+## Overview
+
+The Analyze Experiment command examines traces logged in an MLflow experiment to automatically discover:
+
+- **Operational Issues**: Errors, timeouts, rate limiting, authentication failures, and performance bottlenecks
+- **Quality Issues**: Overly verbose responses, inconsistent outputs, repetitive content, and inappropriate response formats
+- **Success Patterns**: What's working well, effective tool usage, and high-quality interactions
+- **Performance Metrics**: Latency distributions, success rates, and error patterns
+
+The tool generates a detailed markdown report with specific trace examples, quantitative evidence, and actionable recommendations for improvement.
+
+## Usage
+
+The Analyze Experiment functionality is available through two methods:
+
+<Tabs>
+<TabItem value="mcp" label="MCP" default>
+
+### Using MCP
+
+If you have MLflow's MCP server configured, you can simply run:
+
+```
```

Comment:
don't forget language indicator for formatting `bash`

Reply:
done